webapp/src/components/header/settings-dialog/SettingsDialog.tsx

```diff
@@ -93,6 +93,20 @@ export const SettingsDialog: React.FC<ISettingsDialogProps> = ({ open, closeDial
                                     })}
                                 </AccordionPanel>
                             </AccordionItem>
+                            <Divider />
+                            <AccordionItem value="about">
+                                <AccordionHeader expandIconPosition="end">
+                                    <h3>About</h3>
+                                </AccordionHeader>
+                                <AccordionPanel>
+                                    <Body1 color={tokens.colorNeutralForeground3}>
+                                        Frontend version: {process.env.REACT_APP_SK_VERSION ?? "-"}<br/>
```

Comment:
instead of defining a new env variable for the version of the UX, I think we can pull this from the `version` field in `package.json`.

Reply:
Intriguing idea. Are you proposing to do a second commit for every merge? Or intercepting merge requests and changing the version number then?

Reply:
my initial thought was the latter, to move the onus of keeping up with the version away from the developer. 
but I see that the sole purpose of this variable is to keep track of what's currently deployed, and in that case we shouldn't be touching any of the actual files.

resolving 😄